lib/workers/repository/reconfigure/index.ts

```diff
@@ -17,15 +33,117 @@ export async function checkReconfigureBranch(
     return;
   }
 
+  const context = config.statusCheckNames?.configValidation;
   const reconfigureBranch = getReconfigureBranchName(config.branchPrefix!);
   const branchExists = await scm.branchExists(reconfigureBranch);
 
-  // this is something the user initiates, so skip if no branch exists
+  // this is something the user initiates, skip if no branch exists
   if (!branchExists) {
     logger.debug('No reconfigure branch found');
     deleteReconfigureBranchCache(); // in order to remove cache when the branch has been deleted
     return;
   }
 
-  await validateReconfigureBranch(config);
+  const existingPr = await platform.findPr({
+    branchName: reconfigureBranch,
+    state: 'open',
+    includeOtherAuthors: true,
```

Comment:
doesn't work on gotta and other platforms because we only have our own PRs cached

Reply:
We could add this support as per user requests.

Reply:
Yes, I don't think we require ther Prs from users except the reconfigure PR. 

Github has this implemented neatly. I can replicate the same for other platforms. Should I do that before hand?

Reply:
https://github.com/renovatebot/renovate/pull/35065

Reply:
you need to pass `targetBranch` to get it working on gitea